src/util/keyutil.ts

```diff
@@ -114,6 +123,79 @@ class KeybindManager {
         this.keyDescriptionsMap = newKeyDescriptions;
     }
 
+    prettyPrintKeybind(keyDescription: string): string {
+        let keyPress = parseKeyDescription(keyDescription);
+        let returnString = "";
+        if (keyPress.mods.Cmd) {
+            returnString += "⌘";
```

Comment:
should I make these platform specific? I was wondering if I should make it the windows key on linux, idk

Reply:
ya, that's a good idea to make them platform specific.